internal/util/K8sUtil.go

```diff
@@ -146,6 +146,15 @@ func (impl K8sUtil) UpdateConfigMap(namespace string, cm *v1.ConfigMap, clusterC
 	}
 }
 
+func (impl K8sUtil) CreateConfigMapFast(namespace string, cm *v1.ConfigMap, client *v12.CoreV1Client) (*v1.ConfigMap, error) {
```

Comment:
any logic behind `Fast` suffix?

Reply:
nothing inside, just passing core client functions to functions, not creating in every call.